registry/access_control/README.md

```diff
@@ -0,0 +1,64 @@
+# Feathr Registry Access Control Gateway Specifications
+
+## Registry API with Access Control Gateway
+
+**Access Control Gateway** is an access control **Plugin** component of feature registry API. It can work with different type of backend registry. When user enables this component, registry requests will be validated in a gateway as below flow chart:
+
+```mermaid
+flowchart TD
+    A[Get Registry API Request] --> B{Is Id Token Valid?};
+    B -- No --> D[Return 401];
+    B -- Yes --> C{Have Permission?};
+    C -- No --> F[Return 403];
+    C -- Yes --> E[Call Registry API*];
```

Comment:
>Registry [](http://example.com/codeflow?start=24&length=8)

Call it downstreaming api might make more sense as new APIs like metrics, jobs will come soon

Reply:
Updatd to "Downstream API"